backend/custom_plugins/templates/custom_plugins/blurb.html

```diff
@@ -1,5 +1,6 @@
 {% load cms_tags %}
-<div class="blurb">
+
+<div class="blurb {{instance.type}}">
```

Comment:
Please use both `blurb` and `blurb-link`/ `blurb-button` as classes, so that output is either `<div class="blurb blurb-link">` or `<div class="blurb blurb-button">`.